app/templates/brief.html

```diff
@@ -70,16 +70,36 @@
     </div>
   </div>
 {% elif brief.status == 'live' %}
-  <div class="grid-row">
-    <div class="column-one-third">
-      {% with
-         url = "/suppliers/opportunities/{}/responses/create".format(brief.id),
-         label = "Start application"
-      %}
-        {% include "toolkit/link-button.html" %}
-      {% endwith %}
+  {% if 'NEW_SUPPLIER_FLOW' is active_feature %}
+    <div class="grid-row">
+      <div class="column-two-thirds">
+        <h2 class="summary-item-heading">Apply for this opportunity</h2>
+        <p class="dmspeak">To apply, you must give evidence for all the essential and nice-to-have skills and experience you have.</p>
+      </div>
     </div>
-  </div>
+    <div class="grid-row">
+      <div class="column-one-third">
+        {% with
+           url = "/suppliers/opportunities/{}/responses/create".format(brief.id),
+           label = "Apply",
+           advice = True
```

Comment:
Optional improvement but could maybe do with a small comment here to say what advice is doing (group this button with the above advice).